src/components/tables/test/utils.test.ts

```diff
@@ -0,0 +1,40 @@
+import { dateSorter, renderActivities, renderSiteIdAsLink } from '../utils';
+import { render, screen } from '@testing-library/react';
+
+test('dateSorter', () => {
+  const date1: Date = new Date();
+  const date2: Date = new Date(date1.valueOf() - 1);
+  const date3: Date = new Date(date1.valueOf() - 21);
+
+  expect(dateSorter(date1, date2)).toEqual(1);
+  expect(dateSorter(date2, date1)).toEqual(-1);
+  expect(dateSorter(date3, date2)).toEqual(-20);
+  expect(dateSorter(date1, date1)).toEqual(0);
+});
+
+test('renderSiteIdAsLink', () => {
+  const siteId = 101;
+  render(renderSiteIdAsLink(siteId));
+  const linkElement = screen.getByText('101');
+  expect(linkElement).toBeInTheDocument();
+  expect(linkElement).toHaveAttribute('href', '/tree/101');
+});
+
+test('renderActivities', () => {
+  render(renderActivities(['watered']));
+  let wateredTags = screen.getAllByText('watered');
+  expect(wateredTags.length).toEqual(1);
+
+  render(renderActivities(['watered', 'mulched']));
```

Comment:
❓ Is it good practice/ok to call `render` multiple times in one test like this?

Reply:
The reason (that I could find) why it might be bad to call `render` multiple times is that it creates a new element each time, so if you're expecting a simple re-render you'd end up with multiple elements. In this case, that's what we want so it's fine. Still, I split this into two tests (to be more precise on what was being tested) which happens to remove the need for multiple renders.